packages/mobile/e2e/src/usecases/ResetAccount.js

```diff
@@ -0,0 +1,44 @@
+import { enterPinUiIfNecessary, waitForElementId, sleep } from '../utils/utils'
+import { SAMPLE_BACKUP_KEY } from '../utils/consts'
+
+export default ResetAccount = () => {
+  it('Reset Account by doing the Account Key quiz', async () => {
+    // This test has very high flakiness on Android. I did my best to fix it, but
+    // locally it works every time but on CI it fails 50%+ of the time.
+    // TODO: Keep investigating the source of flakiness of this test on Android.
+    if (device.getPlatform() === 'android') {
+      return
```

Comment:
I am hoping that upgrading our Circle instance will improve on this. I believe @annakaz is looking into that.